schema-admin.graphql

```diff
@@ -382,4 +386,8 @@ type Mutation {
   Creates a Label.
   """
   createLabel(name: String!): Label!
+  """
+  Updates a Label.
```

Comment:
Maybe worth saying that updates a label that is not assigned to any collection yet?